### ISSUE TITLE
Current Taskwarrior alias does not work?

### ISSUE THREAD

**Issue by manoelpqueiroz** [ohmyzsh/ohmyzsh#11257]
### Describe the bug

I would just like to make sure that the current implementation of Taskwarrior's [plugin](https://github.com/ohmyzsh/ohmyzsh/blob/master/plugins/taskwarrior/taskwarrior.plugin.zsh) should in fact not work due to the lack of ticks.

Currently, the plugin sets a general alias to reduce the command with `alias t=task`. However, my Zsh shell does not recognise `t` as a command after sourcing `zshrc`.

Should it be `alias t='task'` instead? Does the lack of ticks render the alias unusable? One thing that confuses me if that the plugin also implements an autocompletion command right after setting the alias, which is working fine with my installation.

### Steps to reproduce

1. Enable plugin `taskwarrior`
2. Source `.zshrc`
3. Run command `t`

### Expected behavior

By simply typing `t` at the terminal, a list of current tasks should show up at the terminal.

### OS / Linux distribution

Arch Linux

### Zsh version

5.9

### Oh My Zsh version

master (18e7e5d0)

### Terminal emulator

Konsole

**Comment by rwmitchell** [comment]
Have you tried entering
```
alias t='task`
```
at the cmdline to see if it started working?  You can set aliases right in the current shell without going thru plugins or .zshrc files.

Have you installed taskwarrior ?  Is it installed in your PATH ?

**Comment by manoelpqueiroz** [comment]
Yes, I have Taskwarrior installed and setting the alias for my session worked out. I'll open a pull request to correct the plugin's behaviour.

**Comment by rwmitchell** [comment]
hmmm, your results were unexpected, at least on my system, the single quotes around task is not required.

**Comment by manoelpqueiroz** [comment]
Now that's interesting. Any recommendations as to how to debug and try to reach the root-cause?

**Comment by rwmitchell** [comment]
The single quotes are necessary when the thing being aliased includes multiple items, in this case there is only one.
You can test this on your system by repeating the previous test, but without the single quotes.

When I ```source taskwarrior.plugin.zsh``` t returns "command not found: task", which is correct as I don't have it installed, but it does show the full command.  Something in your .zshrc must be overriding  't'

What does ```which t``` report after sourcing .zshrc ?

**Comment by manoelpqueiroz** [comment]
@rwmitchell, sorry for the late reply. But in my case, even by manually sourcing `.zshrc`, it does not recognise `t` as a command.

I'm using P10K as my theme, I did try fumbling through `p10k.zsh` to see if anything was interfering, but didn't manage to find out anything.

**Comment by rwmitchell** [comment]
Sorry, what does
which task
report ?

**Comment by manoelpqueiroz** [comment]
`which task` points to `/usr/bin/task`, as expected.

However, `which t` outputs "t not found", thus `alias t` return empty.

I found this odd since the [plugin](https://github.com/ohmyzsh/ohmyzsh/blob/master/plugins/taskwarrior/taskwarrior.plugin.zsh) specificies the `t` alias, but doesn't show up for me. Does your installation work correctly?

**Comment by carlosala** [comment]
This is not reproducible to me. Could you please create a trace of the zsh startup with the command below, and attach the file (omz-debug.log) here for inspection
```sh
zsh -xvlic exit &> ~/omz-debug.log
```
Thanks!

**Comment by mcornella** [comment]
Look whether you have an `unalias -m '*'` line in your .zshrc files, which would be visible in the omz-debug.log generated from https://github.com/ohmyzsh/ohmyzsh/issues/11257#issuecomment-1304050087.

**Comment by carlosala** [comment]
Closing as it's probably a matter of removing the alias in some part of the code.
